src/Nethermind/Nethermind.RunnerSpawner/Program.cs

```diff
@@ -118,7 +119,7 @@ private static ProcessWrapper CreateAppConsole(string path, string baseDbPath, s
                 arguments = $"{arguments} --baseDbPath {baseDbPath}";
             }
             process.StartInfo.Arguments = arguments;
-            process.StartInfo.UseShellExecute = true;
+            process.StartInfo.UseShellExecute = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
cool

Reply:
It took me a while to figure out, why it doesn't work on my *NIX.

Reply:
I wonder if this is the same thing that fails when you launch it from git bash

Reply:
Could be, need to give it a try.